engine/framework/model/master.go

```diff
@@ -40,6 +43,56 @@ const (
 	// extend the status code here
 )
 
+// MasterMetaExt stores some attributes of job masters that do not need
```

Comment:
Does sqlite mock client support this json?

Reply:
It seems working for now. We may encounter problem if we were to use advanced JSON functions, but they do not seem needed for now.